### ISSUE TITLE
The model output appears to be random when I use my image？

### ISSUE THREAD

**Issue by hustCYQ** [likelyzhao/vit-pytorch#2]
My code:

`if __name__ == '__main__':
    import torch
    import cv2
    input_size = 224
    img = cv2.imread("OIP-C.jpg")
    img = cv2.resize(img,(224,224))
    img = rearrange(img, 'h w c -> c h w')
    img = repeat(img,'c h w -> b c h w',b=1)
    print(img.shape)
    img = torch.Tensor(img).cuda()

    v = VIT_B16_224()

    v.load_state_dict(torch.load('imagenet21k+imagenet2012_ViT-B_16-224.pth'))
    v.cuda()
    preds = v(img) # (1, 1000)
    preds = torch.softmax(preds,-1)
    print(preds.flatten())
    print(torch.max(preds.flatten()))`

and the output is different each time.
